incubator/hnc/pkg/reconcilers/hierarchy_config.go

```diff
@@ -423,28 +423,34 @@ func (r *HierarchyConfigReconciler) enqueueAffected(log logr.Logger, reason stri
 }
 
 func (r *HierarchyConfigReconciler) writeInstances(ctx context.Context, log logr.Logger, oldHC, newHC *api.HierarchyConfiguration, oldNS, newNS *corev1.Namespace) (bool, error) {
-	ret := false
-	if updated, err := r.writeHierarchy(ctx, log, oldHC, newHC); err != nil {
+	isDeletingNS := !newNS.DeletionTimestamp.IsZero()
+	updated := false
+	if up, err := r.writeHierarchy(ctx, log, oldHC, newHC, isDeletingNS); err != nil {
 		return false, err
 	} else {
-		ret = ret || updated
+		updated = updated || up
```

Comment:
isn't `updated` always false at this point?

Reply:
Yes, I just wanted it to look the same as the one below :)

Reply:
Thats what I figured, thanks!